### ISSUE TITLE
RasperryPi

### ISSUE THREAD

**Issue by 10ra** [TelluIoT/ThingML#233]
How to install ThingML in RaspberryPi. can u give a detailed procedure for it.

can u give the "Compiling ThingML Generated Code" in detailed step. It is very hard to follow. Can you provide with screenshots of the istalation and compilation steps.

**Comment by jakhog** [comment]
Hello @10ra ! I see now that the documentation in the README, could be a bit clearer.

But first, maybe we should figure out what you want do do. Is it:
1) Install the ThingML tools (Eclipse + Plugins) to write a ThingML model and compile it
2) Use the tools on your computer, to write a ThingML model, and compile it so that it can run on a RPi
?
